tools/moFF/moff.xml

```diff
@@ -421,6 +421,8 @@ It currently uses a Go library to directly extract data from Thermo Raw spectrum
 eliminating the need for conversions from other formats.
 Moreover, moFF also allows one to work directly with mzML files.
 
+This is moFF version @VERSION@.
```

Comment:
Don't do it like this. Use https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-version-command

Reply:
I thought about this, but I don't think moff has a `moff.py --version`, as far as I can tell. Is this way okay or is there a conda workaround?

Reply:
No conda workaround. But you could echo "@VERSION@". Which is also bad but better then nothing.
moff should implement the version ... :)